spec/builtins/mv.spec.js

```diff
@@ -1,65 +1,105 @@
 import { CommandValidator } from "../../src/cmd-validator.js";
 
-describe("mv command validation", () => {
+describe("rm command validation", () => {
```

Comment:
rm should not be here. 

Go through the rest of the files, ensure the right command is in the right spec file, remove duplicate test cases.